elastic/helm-charts
# fix 854 68

- [release] Initiate 7.7 branch (#541)
- [meta] fix ci jobs
- [jenkins] Fix Google cluster name (#571)
- [ci] Fix triggering upstream cleanup job (#578)
- Merge pull request #567 from jmlrt/metricbeat-improvements
- Merge pull request #568 from jmlrt/filebeat-improvements
- [doc] Update links to 7.7 branch (#579)
- Merge pull request #580 from jmlrt/helm-2-16-6
- [script] Update bumper script to handle SNAPSHOT versions (#581)
- [ci] Fix multijob configuration for testing staging artifacts (#582)
- Merge pull request #583 from Conky5/use-different-image-for-random-key
- Merge pull request #572 from jmlrt/split-metricbeat-values
- Merge pull request #585 from jmlrt/metricbeat-hostnetwork
- Merge pull request #590 from marcostvz/patch-1
- Merge pull request #584 from michelesr/master
- Merge pull request #522 from domgoodwin/helm-default-patch
- Merge pull request #586 from jmlrt/readiness-to-503
- Merge pull request #569 from cartonalexandre/master
- Merge pull request #591 from jmlrt/logstash-http-host-var
- Merge pull request #593 from jmlrt/doc-improvements (#599)
- FAQ and examples improvements (#598) (#600)
- [meta]  bump version on examples readme (#601)
- initiate 6.8 branch (#602)
- [helm] bump helm version to 2.16.7 (#607)
- 6.8.9 Release (#617)
- [metricbeat] fix deployment upgrade by removing chart label from .spec.selector.matchLabels (#622)
- [meta] bump to 6.8.10-SNAPSHOT (#618)
- Fix configchecksum not being set (#634)
- Fix values links in examples/multi/README.md (#639)
- [kibana] String/YAML conditions for `.Values.{extraContainers,extraInitContainers}` (#637)
- [elasticsearch] Fix issue with `readinessProbe` causing outages (#638)
- [metricbeat] use deprecated kube-state-metrics config when existing (#624)
- Elasticsearch secret mountmode (#596)
- [kibana] Add extensible label support (#555)
- [meta] add support for k8s 1.16 (#635)
- 6.8.10 Release (#651)
- [meta] bump to 6.8.11-SNAPSHOT (#658)
- [helm] bump helm version to 2.16.8 (#657)
- Elasticsearch: do not include heritage selector (#437)
- [meta] fix release install commands (#662)
- [apm-server] increase memory limit (#664)
- [elasticsearch] Disable service links to prevent very long startup times (#542)
- [apm-server] allow customizing probes (#671)
- [elasticsearch] update kind example for version >= 0.7.0 (#669)
- [helm] bump helm version to 2.16.9 (#667)
- [logstash] add security example (#392)
- podSecurityContext.runAsUser needs to be nulled as well for Openshift (#655)
- [meta] disable stackdriver for gke test clusters (#687)
- [meta] add back beta notices for 6.8 (#692)
- Add ServiceAccount annotations (#686)
- [logstash] fix version in security test (#696)
- Elasticsearch: set PVC labels through setting all StatefulSet labels to its volumeClaimTemplate (#665)
- [docs] Improve visibility of 6.8 and 7.x releases (#690)
- [doc] fix copy-paste errors
- [logstash] add headless service for statefulset (#695)
- [filebeat] add permission to list nodes (#704)
- [testing] Ensure that the 'namespace' is specified in Helm command (#715)
- feat(metricbeat): annotation support for daemonset and deployment (#713)
- [apm-server] Add to test matrix (#708)
- Support creating k8s helm secrets for logstash helm chart #705 (#712)
- [filebeat] document probe workaround for kafka output (#699)
- Metricbeat daemonset deployment optional (#716)
- Make kube-state-metrics optional for installation (#387)
- [elasticsearch] _helpers.tpl - elasticsearch.endpoints to use elasticsearch.uname (#670)
- [logstash] remove expected logstash host and name assertion (#724)
- [logstash] Restart the logstash pod when the secrets have changed (#723)
- [elasticsearch] Update test image pull policy. (#727)
- [Kibana] Add loadbalancerIP to service spec in kibana (#726)
- [doc] fix some links (#737)
- [elasticsearch] remove initial_master_nodes test (#740)
- Update README.md (#749)
- 6.8.11 Release (#753)
- [meta] add slack notifications to CI jobs (#760)
- [doc] update doc links (#758)
- [meta] add helm 3 beta support (#759)
- [meta] remove branch specific staging jobs (#736)
- [release] bump 6.8 branch to 6.8.12-SNAPSHOT (#762)
- 6.8.12 release (#787)
- Bump version to 6.8.13-SNAPSHOT  (#790)
- [meta] drop gke 1.14 tests (#798)
- [helm] bump helm version to 2.16.10 (#797)
- Fix typo in FAQ (#744)
- Missing deletion of "elastic-certificate-crt" (#752)
- Remove duplicate "initialDelaySeconds" field (#763)
- [logstash] Fix headless service ports spec (#776)
- [metricbeat] Add missing labels for deployment (#770)
- Fix serviceAccount for APM server (#786)
- [metricbeat] Support secrets (#778)
- [elasticsearch] add loadBalancer externalTrafficPolicy option (#810)
- Include pre-releases in the semver range. (#729)
- [elasticsearch] fix secrets names in examples (#811)
- [helm] bump helm version to 2.16.12 (#816)
- [elasticsearch][kibana] disable nss dentry cache (#818)
- support tpl in logstashConfig, logstashPipeline and kibanaConfig (#717)
- [metricbeat] support deployment/daemonset specific metrics (#820)
- [filebeat] introduce dnsConfig values for the containers (#659)
- [logstash] use only httpPort in headless service (#839)
- Added ingress support to the logstash chart (#793)
- [Metricbeat] Dont generate config if not enabled (#767)
- [all] Simplify doc in 6.8 branch (#858)
- 6.8.13 release (#861)
- Remove beta warnings from the 6.8 branch (#868)
- Bump 6.8 branch to 6.8.14-SNAPSHOT (#871)
- [elasticsearch] add coordinator node to multi test (#854)
- [elasticsearch] remove roles unavailable on 6.8
